qa/rgw/store/sfs/build-radosgw.sh

```diff
@@ -114,11 +114,11 @@ _build() {
   if [ "${WITH_TESTS}" == "ON" ] ; then
     # discover tests from build.ninja so we don't need to update this after
     # adding a new unit test
-    # SFS unittests should be named unittest_rgw_sfs*
+    # SFS unittests should be named unittest_rgw_sfs_*
+    # SFS unittests should be named unittest_rgw_s3gw_*
     IFS=" " read -r -a \
-      UNIT_TESTS <<< "$(awk '/build unittest_rgw_(sfs|s3gw)/ {
-                               print $4
-                             }' build.ninja)"
+      UNIT_TESTS <<< "$(grep -E "build unittest_rgw_sfs_|build unittest_rgw_s3gw_" build.ninja \
+                          | awk 'BEGIN {ORS=" "}; {print $4}')"
```

Comment:
Ok, after running both commands I finally understand what you are trying to fix. However, the new command does not pick up on an additional unit test that would be picked up by the original `awk` command: `src/test/rgw/sfs/unittest_rgw_sfs`, because the new command expects an underscore after `_rgw_sfs`.

Reply:
the problem with the original command is that:
1. it is capturing also `unittest_rgw_sfs` and `unittest_rgw_s3gw` directories, we want to capture binaries only (with the trailing underscore we should prevent this).
2. it is separating the captured tokens with a newline, but CI is not working as intended due to that; the new patch splits tokens with a space.